### DISCUSSION TITLE
Issue with saving the template- 'The template isn't updated'

### DISCUSSION THREAD

**Discussion by Stephooooo** [emailjs-com/help#22] [Q&A] [answered]
Hi! please help!

Every time I make changes to my template I am unable to save it. It always throws an alert message citing 'The template Isn't updated' . I really don't know what to do. Kindly assist, Thank you!

**Comment by xr0master** [comment, marked as answer]
Hi, this is a very strange issue. If you are trying to save and there are no changes, it is possible to get this message as there is nothing to update. (We think to change this mechanism as it confuses users a bit). However, if you've made changes and still can't save your template... it will be hard to understand. Since there are no similar complaints from other users, I recommend trying to use the incognito mode without plugins that may affect the correct operation.

**Reply by Stephooooo** [reply]
Hi @xr0master 

Thank you so much for your help! I tried again using Incognito mode and I've managed to save the template successfully!
I can confirm that this works.
